src/uct/ib/rc/verbs/rc_verbs_iface.c

```diff
@@ -242,6 +243,12 @@ static UCS_CLASS_INIT_FUNC(uct_rc_verbs_iface_t, uct_md_h md, uct_worker_h worke
     self->config.max_inline = cap.max_inline_data;
     uct_ib_iface_set_max_iov(&self->super.super, cap.max_send_sge);
 
+    if (self->config.max_inline < sizeof(uct_rc_hdr_t)) {
+        self->fc_desc = ucs_mpool_get(&self->short_desc_mp);
+        ucs_assert_always(self->fc_desc != NULL);
+        hdr        = (uct_rc_hdr_t*)(self->fc_desc + 1);
+        hdr->am_id = UCT_RC_EP_FC_PURE_GRANT;
+    }
```

Comment:
we can set self->fc_desc=NULL if not valid, and in iface d'tor check if it's NULL. this way the assertion ` ucs_assert(iface->fc_desc != NULL);` in FC send would make more sense

Reply:
ok, done
iface->fc_desc == NULL by default by memset of iface object